_security/access-control/api.md

```diff
@@ -1297,6 +1297,91 @@ PATCH _plugins/_security/api/securityconfig
 }
 ```
 
+### Configuration upgrade check
+Introduced 2.13
+{: .label .label-purple }
+
+Checks the configuration bundled with the Security plugin against the existing security configuration, returns if an upgrade can be performed and what resources would be updated.
+
+Between new OpenSearch version new defaults to the security configuration are added.  This endpoint helps cluster operators check if the cluster is missing defaults or has stale definitions of defaults. 
```

Comment:
wdyt about this?

Between new OpenSearch versions, there may be changes to the default security configuration like the addition or modification of roles for plugins included in the default OpenSearch distribution. This endpoint helps cluster operators check if the cluster is out of sync with the defaults.